less-print/viewdetails.less

```diff
@@ -1,5 +1,90 @@
 .view-details-associations, .view-details-comments, .view-details-photos,
-.scroll-top-btn, .view-details-map, .location-static, .badge, .icons-header,
-.logo {
-  display: none;
+.scroll-top-btn, .view-details-map, .badge, .icons-header {
+  display: none !important;
+}
+
+.view-details-title{
+	display: block;
+	font-size: .75em !important;
+	margin-top: 0px;
+	margin-bottom: 10px;
+}
+
+.description , .view-details-info {
+	display: block !important;
+}
+
+.view-details-section {
+	display: block;
+	font-size: .75em !important;
+	margin-left: 0;
+	padding-bottom: 0;
+	width: 100%;
+}
+
+.lead{
+	padding: 0 10px !important;
+}
+
+.heading {
+	padding: 5px 10px !important;
+}
+
+.view-details-section .view-details-informations .accordion {
+    padding: 0;
+}
+
+.document-summary {
+	border: none !important;
+	padding: 5px;
+	margin: 0;
+}
+
+.view-details-title, .view-details-description, .view-details-informations , .view-details-info {
+	border: none !important;
+	display: block;
+	width: 100%;
+}
+
+#orientation-svg {
+	height: 55px !important;
+	margin-left: 0 !important;
+}
+
+.name-icon-value {
+	border-bottom: none !important;
+	min-height: 10px !important;
+	max-width: 50%;
+	display: inline;
+}
+
+.heading {
+	text-decoration: underline;
+}
+
+.heading.closed{
+ 	display: none;
+}
+
+.heading .glyphicon{
```

Comment:
Maybe use another class? `glyphicon` is a generic class for bootstrap icons. Maybe `.heading .glyphicon-menu-down:last-child`?